### PR TITLE
Implemented accessibe app

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What is the purpose of this pull request?

Implemented the basic functionality of the accessiBe app without the `usefulLinks` working

#### How should this be manually tested?

[workspace](https://iaron--storecomponents.myvtex.com/)
[workspace-apps](https://iaron--storecomponents.myvtex.com/admin/apps/vtex.accessibe@1.0.0/setup/)

#### Screenshots or example usage

image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by iaronaraujo** [comment]
Needs `README`

File: manifest.json [hunk 1]
```diff
@@ -14,5 +14,299 @@
...
+        "type": "string",
+        "title": "Interface Lead Color",
+        "default": "#146ff8"
+      },
+      "triggerColor":{
+        "type": "string",
+        "title": "Trigger Button Color",
+        "default": "#146ff8"
+      },
+      "handleJquery": {
```
Comment (klzns) [inline manifest.json]: This should be always true, dont need to expose this settings.

File: manifest.json [hunk 2]
```diff
@@ -14,5 +14,299 @@
...
+          "properties": {
+            "about": {
+              "type": "string",
+              "default": "About",
+              "title": "Label"
+            },
+            "url": {
+              "type": "string",
+              "default": "https://domain.com/about",
+              "title": "Url"
```
Comment (klzns) [inline manifest.json]: Change it to URL

File: pixel/body.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      script.src = 'https://acsbap.com/api/app/assets/js/acsb.js'; script.async = true; script.defer = true;
+      (typeof element === 'undefined' ? document.getElementsByTagName('html')[0] : element).appendChild(script);
+      script.onload = function() {
+          acsbJS.init({
+              statementLink     : '{{settings.statementLink}}',
+              feedbackLink      : '{{settings.feedbackLink}}',
+              footerHtml        : '{{settings.footerHtml}}',
+              hideMobile        : {{settings.hideMobile}},
+              hideTrigger       : {{settings.hideTrigger}},
+              handleJquery      : {{settings.handleJquery}},
```
Comment (klzns) [inline pixel/body.html]: This is always true.

File: pixel/body.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<script>
+  (function(document, tag) {
+      var script = document.createElement(tag);
+      /*var usefulLinks = {{settings.usefulLinks}}
```
Comment (iaronaraujo) [inline pixel/body.html]: @klzns if I remove this comment here and then pass some values to `Useful Website Links` through the app page, this app breaks. Do you have any idea of what I am doing wrong?

Comment (jgfidelis) [reply pixel/body.html]: @klzns se ele passa um array la no admin, quando tentamos acessar `{{settings.usefulLinks}}` ele quebra aqui, é bem estranho. Precisamos de ajuda para debuggar pois não sei como esse settings é magicalmente acessado aqui, parece um problema na hora de parsear o array.

Comment (jgfidelis) [reply pixel/body.html]: Deve ser um problema no builder de pizel, ou seja lá o código que le e parseia o settings

File: pixel/body.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        return acc
+      }, {})
+      var element = document.getElementsByTagName('body')[0];
+      script.src = 'https://acsbap.com/api/app/assets/js/acsb.js'; script.async = true; script.defer = true;
+      (typeof element === 'undefined' ? document.getElementsByTagName('html')[0] : element).appendChild(script);
+      script.onload = function() {
+          acsbJS.init({
+              statementLink     : decodeURIComponent('{{settings.statementLink}}'),
+              feedbackLink      : decodeURIComponent('{{settings.feedbackLink}}'),
+              footerHtml        : '{{settings.footerHtml}}',
```
Comment (jgfidelis) [inline pixel/body.html]: decode this too

File: README.md [hunk 6]
```diff
@@ -6,3 +6,5 @@ accessiBe first-party app.
 
 Open the VTEX App Store and install this app on your store.
 
+Then, open the [accessiBe website](https://accessibe.com/), login with your account and then add your vtex store url to `my websites` in the `Manage Websites` tab.
```
Comment (klzns) [inline README.md]:
```suggestion
Then, open the [accessiBe website](https://accessibe.com/), login with your account and then add your VTEX store url to `my websites` in the `Manage Websites` tab.
```

File: pixel/body.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+                  triggerPositionY  : '{{settings.mobileTriggerPositionY}}',
+                  triggerOffsetX    : {{settings.mobileTriggerOffsetX}},
+                  triggerOffsetY    : {{settings.mobileTriggerOffsetY}},
+                  triggerRadius     : decodeURIComponent('{{settings.mobileTriggerRadius}}')
+              }
+          });
+      };
+  }(document, 'script'));
+  </script>
+
```
Comment (klzns) [inline pixel/body.html]:
```suggestion
```

**Pull request merged**